src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -362,12 +367,12 @@ protected override async Task<RunResult> RunAsyncInternal(string expr, string se
                     if (!NumberIsFloat && e.Message.Contains("value is too large"))
                     {
                         // Serialization test. Serialized expression must produce an identical result.
-                        ParserOptions options = new ParserOptions() { NumberIsFloat = true };
+                        options.NumberIsFloat = true;
                         newValueDeserialized = await engine.EvalAsync(sb.ToString(), CancellationToken.None, options, runtimeConfig: runtimeConfig).ConfigureAwait(false);
                     }
                     else
                     {
-                        throw;
+                        throw e;

```

Comment:
> throw e; [](http://example.com/codeflow?start=23&length=9)

nit - changing to 'throw e' means we lose the callstack of the original exception. 
Can we keep as 'throw' ?

Reply:
Fixed.